extension/src/fileSystem/dvcYaml/dvcYaml.ts

```diff
@@ -0,0 +1,240 @@
+import { load } from 'js-yaml'
+import { Any } from '../../util/object'
+
+export interface DvcYAML {
```

Comment:
This was auto-generated based on the schema, but I had to fix some of the definitions.